src/react/components/SvgRadioButton.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Select, Tooltip } from 'oskari-ui';
+import { Form, Card, Space, Input, Row, Radio } from 'antd';
+import styled from 'styled-components';
+
+const formLayout = {
+    labelCol: { span: 24 },
```

Comment:
What does 24 mean here? Should it be related to the content of the options shown?

Reply:
AntD divided all components to 24 sections so `span: 24` here means that label column of this component is full width.

https://ant.design/components/grid/